state/state.go

```diff
@@ -468,12 +468,6 @@ func (st *State) getSingularLeaseClient() (lease.Client, error) {
 	return client, nil
 }
 
-// ModelTag() returns the model tag for the model controlled by
```

Comment:
If ModelTag() is going, then ModelUUID() should also go as they are equivalent. Is this planned for a followup refactoring?

Reply:
As discussed, this will be done in a followup.